### DISCUSSION TITLE
Oxidus (redesigned) | Palette color generator for Shadcn projects

### DISCUSSION THREAD

**Discussion by JairHuamanBellido** [shadcn-ui/ui#4534] [Show and tell]
Hey folks!

I just wanted to posted a new versión of Oxidus (here is the [last post](https://github.com/shadcn-ui/ui/discussions/1884) I made  ), in this one we include:
- WCAG Contrast color checker
- Customize color of each variable
- Update showcase componentes.

Repository: https://github.com/JairHuamanBellido/Oxidus
Website: https://oxidus.vercel.app/

Hope you like it 🙏.

<img width="1785" alt="image" src="https://github.com/user-attachments/assets/24cc1f81-f714-4df7-84ba-b78732d6ee80">
<img width="1785" alt="image" src="https://github.com/user-attachments/assets/cf3d56ca-38d3-418f-a8c4-ec6fc6a1af4e">
<img width="1785" alt="image" src="https://github.com/user-attachments/assets/03f2af97-92d0-4041-9133-8ebf17d4b1f3">
<img width="1785" alt="image" src="https://github.com/user-attachments/assets/b404e7f3-13bc-4829-ae04-f4ed41a3fbd1">
<img width="1785" alt="image" src="https://github.com/user-attachments/assets/e231f49b-12b5-4e54-8687-f89fe558b4e5">

**Comment by meetqy** [comment]
Ai generator shadcn/ui [ai generator shadcn/ui theme](ipalettes.com/theme/shadcn)

Youtube demo: https://youtu.be/TBNbLyCZSmE?si=faqCJKG78osFK7s2

![179shots_so](https://github.com/user-attachments/assets/1169342f-cf2e-49a9-8fe9-e3f8b8b67bff)

<img width="1317" alt="image" src="https://github.com/user-attachments/assets/dffaf112-20f5-419c-b527-c3169f0634f9" />
